src/main/scala/io/iohk/ethereum/testmode/TestModeComponentsProvider.scala

```diff
@@ -77,8 +77,8 @@ class TestModeComponentsProvider(
       blockchain,
       blockchainReader,
       evmCodeStorage,
-      blockchainConfig,
-      consensus(blockchainConfig, sealEngine).blockPreparator
+      consensus(blockchainConfig, sealEngine).blockPreparator,
```

Comment:
I'm wondering if we should put the seal engine part inside the blockchain config.

Reply:
I think it's for test mode only.